<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripPlanner</class>
 <widget class="QDialog" name="TripPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BeginSaddlebackTrip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>All Campuses Starting from Saddleback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BeginASUTrip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Shortest Trip from ASU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BeginCustomTrip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Trip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BeginUCITrip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>All Campuses From UCI</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
